ocs_ci/ocs/constants.py

```diff
@@ -629,3 +629,6 @@
 
 # gather bootstrap
 GATHER_BOOTSTRAP_PATTERN = 'openshift-install gather bootstrap --help'
+
+DEFAULT_AWS_REGION = 'us-east-2'
```

Comment:
Should this be in `defaults.py` instead of `constants.py`?

Reply:
Thanks, just changed to a new defaults file under cleanup dir. The reason for the new file is that these defaults are not related to the OCS, but rather to the cleanup CLI